commands/sampledata.go

```diff
@@ -0,0 +1,840 @@
+	}
+
+	printer.PrintT("Sampledata successfully processed", job)
+
+	return nil
+}
+
+//nolint:gocyclo
+func sampledataCmdF(c client.Client, command *cobra.Command, args []string) error {
+	seed, _ := command.Flags().GetInt64("seed")
+	bulk, _ := command.Flags().GetString("bulk")
+	teams, _ := command.Flags().GetInt("teams")
+	channelsPerTeam, _ := command.Flags().GetInt("channels-per-team")
+	users, _ := command.Flags().GetInt("users")
+	deactivatedUsers, _ := command.Flags().GetInt("deactivated-users")
+	guests, _ := command.Flags().GetInt("guests")
+	teamMemberships, _ := command.Flags().GetInt("team-memberships")
+	channelMemberships, _ := command.Flags().GetInt("channel-memberships")
+	postsPerChannel, _ := command.Flags().GetInt("posts-per-channel")
+	directChannels, _ := command.Flags().GetInt("direct-channels")
+	postsPerDirectChannel, _ := command.Flags().GetInt("posts-per-direct-channel")
+	groupChannels, _ := command.Flags().GetInt("group-channels")
+	postsPerGroupChannel, _ := command.Flags().GetInt("posts-per-group-channel")
+	profileImagesPath, _ := command.Flags().GetString("profile-images")
+	withAttachments := profileImagesPath != ""
+
+	if teamMemberships > teams {
+		return fmt.Errorf("you can't have more team memberships than teams")
+	}
+	if channelMemberships > channelsPerTeam {
+		return fmt.Errorf("you can't have more channel memberships than channels per team")
+	}
+
+	if users < 6 && groupChannels > 0 {
+		return fmt.Errorf("you can't have group channels generation with less than 6 users. Use --group-channels 0 or increase the number of users")
+	}
+
+	var bulkFile *os.File
+	var tmpDir string
+	var err error
+	switch bulk {
+	case "":
+		tmpDir, err = ioutil.TempDir("", "mmctl-sampledata-")
+		if err != nil {
+			return fmt.Errorf("unable to create temporary directory")
+		}
+		defer os.RemoveAll(tmpDir)
+
+		if withAttachments {
+			if err = os.Mkdir(filepath.Join(tmpDir, attachmentsDir), 0755); err != nil {
+				return fmt.Errorf("cannot create attachments directory: %w", err)
+			}
+		}
+
+		bulkFile, err = os.Create(filepath.Join(tmpDir, "import.jsonl"))
+		if err != nil {
+			return fmt.Errorf("unable to open temporary file: %w", err)
+		}
+	case "-":
+		bulkFile = os.Stdout
+	default:
+		bulkFile, err = os.OpenFile(bulk, os.O_RDWR|os.O_CREATE, 0755)
+		if err != nil {
+			return fmt.Errorf("unable to write into the %q file: %w", bulk, err)
+		}
+	}
+
+	profileImages := []string{}
+	if profileImagesPath != "" {
+		var profileImagesStat os.FileInfo
+		profileImagesStat, err = os.Stat(profileImagesPath)
+		if os.IsNotExist(err) {
+			return fmt.Errorf("profile images folder doesn't exists")
+		}
+		if !profileImagesStat.IsDir() {
+			return fmt.Errorf("profile-images parameters must be a folder filepath")
+		}
+		var profileImagesFiles []os.FileInfo
+		profileImagesFiles, err = ioutil.ReadDir(profileImagesPath)
+		if err != nil {
+			return fmt.Errorf("invalid profile-images parameter")
```

Comment:
any reason not to wrap?
```suggestion
			return fmt.Errorf("invalid profile-images parameter: %w", err)
```